UKHO.PeriodicOutputService/UKHO.BESS.ConfigurationService/BessConfigurationServiceJob.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System.Diagnostics.CodeAnalysis;
+using Microsoft.Extensions.Logging;
+using UKHO.BESS.ConfigurationService.Services;
+using UKHO.PeriodicOutputService.Common.Helpers;
+using UKHO.PeriodicOutputService.Common.Logging;
+
+namespace UKHO.BESS.ConfigurationService
+{
+    [ExcludeFromCodeCoverage]
+    public class BessConfigurationServiceJob
```

Comment:
Why has the capitalisation changed on 'Bess'? Wasn't 'BESS' the correct capitalisation as its an acronym?

Reply:
keep it consistent with existing code.